posthog/tasks/scheduled.py

```diff
@@ -86,6 +87,8 @@ def setup_periodic_tasks(sender: Celery, **kwargs: Any) -> None:
     # Heartbeat every 10sec to make sure the worker is alive
     add_periodic_task_with_expiry(sender, 10, redis_heartbeat.s(), "10 sec heartbeat")
 
+    add_periodic_task_with_expiry(sender, 20, start_poll_query_performance.s(), "20 sec query performance heartbeat")
```

Comment:
Ah, I was suggesting expire, but we have that here already via `add_periodic_task_with_expiry`... not sure it's needed for the individual child tasks then.